src/canvas/PathIntersection.tsx

```diff
@@ -74,7 +69,7 @@ export class PathIntersection {
        and creating a line segment to check for an intersection on. */
     private static _findIntersectionOfLineSegmentAndPath = (thisLineSeg: LineSegment, path: PatternPath): IIntersection | null => {
         // Threshold for checking if a point is on a line. Range from 0 to 1, with 0 being the tightest and 1 being the loosest.
-        const THRESHOLD = .1;
+        const THRESHOLD = .01;
```

Comment:
Why did you make the threshold smaller?

Reply:
While drawing, the algorithm would find an intersection and snap with a path and stop drawing way before we actually touched the path with the mouse. I found that it was because of that threshold that was too loose.

Reply:
Cool, thanks!